blocks/init/src/Blocks/components/video/manifest.json

```diff
@@ -74,6 +87,24 @@
 				"value": "none",
 				"icon": "none"
 			}
+		],
+		"videoSubtitleTrackKind": [
+			{
+				"label": "Subtitles",
+				"value": "subtitles"
+			},
+			{
+				"label": "Captions",
+				"value": "captions"
+			},
+			{
+				"label": "Descriptions",
+				"value": "descriptions"
+			},
+			{
+				"label": "Chapters",
+				"value": "chapters"
+			}
 		]
 	}
-}
+}
```

Comment:
I suggest you enable "Insert an empty line at the end of file." in VS Code. One less thing to worry about 😄